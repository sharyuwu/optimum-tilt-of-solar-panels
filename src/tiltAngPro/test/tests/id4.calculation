id4_1
43.250943
id4_2
-43.250943
id4_3
-89.250943
id4_4
89.250943