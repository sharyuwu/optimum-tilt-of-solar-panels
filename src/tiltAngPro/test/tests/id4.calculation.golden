.calculation
id4_1
Input 23.250943 Output 23.250943 Absolute Erros = 0.0
id4_2
Input -23.250943 Output -23.250943 Absolute Erros = 0.0
id4_3
Input -69.250943 Output -69.250943 Absolute Erros = 0.0
id4_4
Input 69.250943 Output 69.250943 Absolute Erros = 0.0
